database/gui_repo_worker.py

```
@@ -371,6 +371,10 @@ def get_complete_metadata(self, primary_id: str) -> MetadataInfo:
 
         title: str = comic_info[1]
         series: str = comic_info[2]
+        if not (title or series):
+            raise ValueError(
+                f"Comic {primary_id} has missing title or series in database"
+            )
```

Ruff (0.15.17)

[warning] Avoid specifying long messages outside the exception class

(TRY003)